### DISCUSSION TITLE
Show more documentation for overloads when accessing autocomplete

### DISCUSSION THREAD

**Discussion by awa5114** [waf/CSharpRepl#300] [Q&A] [answered]
Is there currently a way to show more documentation in the commandline for a specific overload? Consider the following:

![image](https://github.com/waf/CSharpRepl/assets/24267968/4d806cfd-929a-438f-9585-a89c554aae2d)

The documentation shows the first argument (`type) of the `Add` method and a description, but doesn't show the second one (`attributes`).

Is there a way to "scroll down" such that I can see the rest of the documentation for that particular overload? Thanks in advance

**Comment by kindermannhubert** [comment, marked as answer]
It's the same as in Visual Studio.
To see documentation of the next param, just write `,` and you'll see its documentation.

![image](https://github.com/waf/CSharpRepl/assets/11704036/c0d156f1-dab1-4610-9526-e8c4f71d71be)

Or just this:
![image](https://github.com/waf/CSharpRepl/assets/11704036/773ec8e7-0a52-4886-a43c-7f51bd2812ef)

You can move the caret between commas and see docs for different params:
![image](https://github.com/waf/CSharpRepl/assets/11704036/b7d5e459-d716-4d8e-85c7-94fddbfaf6a2)

I'm not aware of any other possibility to "scroll" through params via keyboard in VS.

**Reply by awa5114** [reply]
That works for me. Thanks.
